### ISSUE TITLE
New Maintainer

### ISSUE THREAD

**Issue by Taar** [convert-units/convert-units#171]
> This post is my attempt to be as transparent as possible.

Hello folks, my name is Randy! I'm introducing myself today because @ben-ng has passed on the role of maintainer for this project to me. Before I go into more details, I would first like to thank @ben-ng for all the work he's put into this project and for giving me this opportunity to give back to the community! I look forward to working with all the contributors of this project! Thanks!

## Immediate Goals

All the immediate goals are focused on getting this project to a point where it's easier for me to maintain and for folks like yourselves to contribute to it.

- Update out of date packages
- Add Github actions to help with reviewing PRs (test runner, linting, etc)
- Add modern code style (using prettier and eslint to enforce it)

Once this is done, I'll start reviewing existing pull requests and outstanding issues.

**Comment by dmbarry86** [comment]
This is good to see. When do you expect to have everything in order and to start reviewing pull requests? I ask because I am interested in using this library for a project but I will need to add some additional conversions. Some of these are covered by existing pull requests and some are not so I would need to create them.

**Comment by Taar** [comment]
@dmbarry86 I've completed the immediate goals listed above and started reviewing pull requests, though, it's going to take some time to review them all. At this time I recommend creating a PR with all of the conversions you want supported regardless of whether or not they are cover by another PR. I recommend this because there is no guarantee that the authors of outstanding PRs will respond to my comments or requests for changes. Unfortunately, since this project hasn't been updated for roughly three years, it's likely that those authors have lost interest in contributing to this project.

Since I haven't had a chance to add a contributing document I'll highlight some of my expectations below:
- Create test cases for all new conversions
- Existing test cases must pass
- No Eslint errors
- Before submitting a PR the code needs be ran through prettier

Please let me know if you have any questions or concerns. I look forward to seeing your PR!

**Comment by rlueder** [comment]
@Taar do you need help maintaining the project? I have some free time in the coming months and would be able to help sort/close open issues and work on some additions to the library. A few tickets I'd be interested in helping with:
#235, #216, #213, #212, #166, #146, #125, #109, #100, #57
